File: barcosEnContienda/src/gestorAplicacion/Personas/CapitanNaval.java
package gestorAplicacion.Personas;



public class CapitanNaval extends Capitanes {
	public CapitanNaval(String apodo, int liderazgo, int elocuencia) {
		super(apodo, liderazgo, elocuencia);
	}
	
	public void yells() {
		System.out.print("Por el honor!\n");
	}
}

File: practica-2-grupo-1-equipo-7-Barcos/src/gestorAplicacion/personas/capitanNaval.py
from gestorAplicacion.embarcaciones.barco import Barco
from gestorAplicacion.personas.capitanes import Capitanes

class CapitanNaval(Capitanes):
    
    def __init__(self,apodo,liderazgo,elocuencia):
        super().__init__(apodo,liderazgo,elocuencia)

    def yells(self):
        return "Por el honor \n"